libs/ui/src/components/navigator/components/component-with-navigation-context/component-with-navigation-context.tsx

```diff
@@ -0,0 +1,9 @@
+import { FC } from 'react';
+
+import { useActiveTokenList } from '../../hooks/use-active-token-list.hook';
+
+export const ComponentWithNavigationContext: FC = () => {
```

Comment:
just run an app from zero in extension and have this 
<img width="956" alt="image" src="https://user-images.githubusercontent.com/35599200/213394495-20e68f0f-8226-4e15-9dbe-c8fc06aeb2a5.png">